modules/migration-mtc-release-notes-1-8-6.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-6_{context}"]
+= {mtc-full} 1.8.6 release notes
+
+[id="technical-changes-1-8-6_{context}"]
+== Technical changes
+
+.Multiple migration plans for a namespace is not supported
+
+{mtc-short} version 1.8.6 and later do not support multiple migration plans for a single namespace.
+
+.VM storage migration
+
+VM storage migration feature changes from Technology Preview (TP) status to being Generally Available (GA).
+
+[id="resolved-issues-1-8-6_{context}"]
+== Resolved issues
+
+{mtc-short} 1.8.6 has the following major resolved issues:
+
+.UI crashes during a namespace search in the migration plan wizard
```

Comment:
Try to use a more descriptive term for "crash", such as "fails" or "ends unexpectedly". IBM SG.